src/main/java/ua/lviv/iot/searcher/Searcher.java

```diff
@@ -0,0 +1,35 @@
+package ua.lviv.iot.searcher;
```

Comment:
You don't have to have packages for every single class :) Think of the folders – do you create separate folders for every file?